packages/runtime/test/errors.test.ts

```diff
@@ -47,13 +48,18 @@ test('crash on runtime error with SyntaxError', async (t) => {
   t.is(error.message, 'SyntaxError: Invalid or unexpected token');
 });
 
-test('crash on runtime error with ReferenceError', async (t) => {
+test.only('crash on runtime error with ReferenceError', async (t) => {
```

Comment:
are we still running this test only?

Reply:
forget. going through by commits first.

Reply:
Ahh don't go by commit, you'll be there for a million years :joy:  This PR has quite a long realtime history and a few pivots along the way. Also my commit history is rarely very neat

Reply:
If it helps we can do a pairing session and I can talk you through some of the key areas, then you can review them in your own time

Reply:
So my review process is to go by commit to understand the entire history and then finally do a broad overview review. 

Yup, the commits don't seem neat.